### ISSUE TITLE
Toot not delete from current cache

### ISSUE THREAD

**Issue by kit-ty-kate** [stom79/mastalab#427]
When deleting a toot that is currently in focus in "conversation mode", when going back to the home timeline, the toot is still there and reloading the timeline does nothing. Only closing the app and reopening it will effectively delete the toot from the cache.

---

Mastalab version: 1.8.7.1

**Comment by stom79** [comment]
Yes. I discarded the "remove" event from the streaming API, I was too lazy :) More seriously, that gives me an idea without using the streaming API, I will simply sent the event as an intent to the main activity (side note for myself).
